src/prisma/schema.prisma

```diff
@@ -2068,6 +2068,7 @@ enum FormFieldType {
   image
   file
   milestone
+  op_project
```

Comment:
i prefer we dont abbreviate and just call it 'optimism_project', but if its a lot of work its not a big deal

Reply:
yeah I like optimism_project_profile